src/redis4net/Redis/IConnectionFactory.cs

```diff
@@ -9,17 +9,21 @@ public class ConnectionFactory : IConnectionFactory
 		private static readonly object Lock = new object();
 
 		private readonly string _hostname;
-		private readonly int _portNumber;
+        private readonly string _password;
+        private readonly int _portNumber;
+        private readonly bool _enableSSL;
```

Comment:
yes, it looks a bit weird here - looks ok in visual studio though - but i can do an extra check